.env.sample

```diff
@@ -0,0 +1,5 @@
+# Heroku Local
+RACK_ENV=development
+PORT=3000
+# Heroku Config Var
+YARN_PRODUCTION=false
```

Comment:
Why do you need this var `YARN_PRODUCTION`?

Also, add the database url variable.

Reply:
`YARN_PRODUCTION` is set in Heroku to install `devDependencies` that include webpack  basically. If I am not set this var heroku will raise an error of webpack dependencies, because by default only install `dependencies`, and  `rails webpacker` uses `webapack` to compile the respective files.

Reply:
See https://github.com/SanGeeky/rails-training/pull/3/files#r614897147